src/PotionItem.js

```diff
@@ -4,11 +4,14 @@ import './PotionItem.css';
 class PotionItem extends Component {
 
   render() {
+
+    const potion = this.props.potionProp;
+    
```

Comment:
Nice work if you want to add the img do <img src={potion.url} alt={potion.title}/>